### PR TITLE
Notifications mobile

### LINKED ISSUE #603: Implement Notifications mobile screen
https://invis.io/UBNS9Y3PK38

### PR DESCRIPTION
**Pull request by valentinewallace**
Closes #603.

### CODE REVIEW

File: src/component/list.js [hunk 1]
```diff
@@ -128,3 +128,34 @@ ListHeader.propTypes = {
...
+export class CardItem extends PureComponent {
+  render() {
+    const { style, ...props } = this.props;
+    return <ListItem style={[cardStyles.item, style]} {...props} />;
+  }
+}
+
+CardItem.propTypes = {
+  style: View.propTypes.style,
+};
```
Comment (tanx) [inline src/component/list.js]: Created a CardItem component that we can reuse.

File: src/view/channel-mobile.js [hunk 2]
```diff
@@ -92,21 +92,13 @@ ChannelList.propTypes = {
...
   item: {
-    flexDirection: 'column',
     alignItems: 'stretch',
-    height: 180,
-    paddingRight: 10,
-    paddingLeft: 10,
-    marginBottom: 15,
-    backgroundColor: color.glasDarker,
-    borderBottomWidth: 0,
-    borderRadius: 7,
```
Comment (tanx) [inline src/view/channel-mobile.js]: I just saw that the new mobile list items all had fixed height. The new CardItem is a bit more flexible.

File: src/component/list.js (hunk 1 above)
Comment (valentinewallace) [reply src/component/list.js]: Looks good :)

**Comment by valentinewallace** [comment]
@tanx Looks great to me! I realized that the border radius wasn't applying to the notification count bubble so I fixed it. If you're okay with my changes feel free to merge.

**Comment by tanx** [comment]
> I noticed that the border radius wasn't applying to the notification count bubble so I fixed it. If you're okay with my changes feel free to merge.

Nice catch! Looks good. Merging :)

**Pull request merged**